services/business/service/tenant-service_test.go

```diff
@@ -71,17 +71,17 @@ var _ = Describe("TenantService Tests", func() {
 		Context("tenant service is instantiated", func() {
 			When("CreateTenant is called without context", func() {
 				It("should return ArgumentError", func() {
-					response, err := sut.CreateTenant(nil, &request)
-					Ω(response).Should(BeNil())
-					assertArgumentError("ctx", "", err)
+					response, _ := sut.CreateTenant(nil, &request)
```

Comment:
You now need to make sure error is returned as nil whenever any service method such as create,read,.... is called

Reply:
fixed